python/pyxir/runtime/tensorflow/ops/tf_l1_basic_nn.py

```diff
@@ -651,3 +651,37 @@ def forward_exec(self, inputs: List[np.ndarray]) -> np.ndarray:
         assert len(inputs) == 1, "Tanh layer expects one input"
         with tf.compat.v1.Session() as sess:
             return sess.run(self.res, feed_dict={self.inpt: inputs[0]})
+
+
+########
+# Dropout #
+########
+
+@rt_register_xlayer_2_tf('Dropout')
+class DroputLayer(rt_layer.BaseLayer, RtLayerTF):
```

Comment:
Can you also add a test for this here: https://github.com/Xilinx/pyxir/tree/master/tests/unit/runtime/tensorflow_rt/ops?